tox.ini

```diff
@@ -3,6 +3,12 @@ minversion = 2.1
 envlist = py35, py36, py37, py38, lint
 skipsdist = True
 
+[pycodestyle]
+count = False
+ignore = E121,E123,E126,E226,E241,E242,E704,E741,W503,W504,W505
```

Comment:
What's the rational for disabling all of these?

Reply:
They are the default disables save for 241 which I need to disable because the plotting program has a kwarg that I need to call that violates that rule.